.github/workflows/ccpp.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       working-directory: ${{ env.dist_dir }}
       run: |
         curl -L -O https://github.com/automaton-network/automaton/releases/download/${{ env.automaton_version }}/automaton-Windows-${{ matrix.configuration }}-${{ matrix.platform }}-${{ env.automaton_version }}.zip
-        unzip automaton-Windows-${{ matrix.configuration }}-${{ matrix.platform }}-${{ env.automaton_version }}.zip
+        unzip automaton-Windows-${{ matrix.configuration }}-${{ matrix.platform }}-${{ env.automaton_version }}.zip -x "include/automaton/*"
```

Comment:
what about the automaton core .a / .lib files? those should be excluded as well, right?